src/proto/robot_config.proto

```diff
@@ -18,7 +18,110 @@ package bowler.robot_config;
 
 option java_multiple_files = true;
 option java_package = "com.commonwealthrobotics.proto.robot_config";
+//next 2
+// a center of mass object
+message CenterOfMass{
+  Pose centroid = 1;
+  double mass_kg = 2;
+}
 
-// Next: 1
+// next 3
+// a file location of a provider of some depency to be injected
+message GitFile {
+  string url = 1;
+  string branch = 2;
+  string file = 3;
+}
+// next 10
+message LimbType{
+  bool isArm = 1;
+  bool isLeg = 2;
+  bool isSteerableWheel = 3;
+  bool isHead = 4;
+  bool isTail = 5;
+  bool isWing = 6;
+  bool isThruster = 7;
+  bool isFin = 8;
+  bool isSpine = 9;
+  bool isFixedWheel=10;
+}
+//next 3
+message Vitamin{
+  string type = 1;
+  string size = 2;
```

Comment:
Can you add comments explaining what these are? `type` sounds like it might be better as an enum, depending on what exactly it means, and `size` doesn't sound like something that should be a string.

Reply:
these are the 2 strings that work on the JCSG Vitamins database, thay are how to access a vitamin.

Reply:
Ill add documentation